README.md

```diff
@@ -41,6 +41,10 @@ Besides the applications that are written in C, it is recommend to use Python
 to write tools or interfaces related to running tests. That allows for simple
 integration with the RF test scripts (and re-usability in general).
 
+## Testing and Changing RIOT Versions
+
+To test a pull request on RIOT go into the RIOT subdirectory and change to the desired state.  Alternatively, the `RIOTBASE=<path_to_riot_directory>` can be used.
```

Comment:
I wouldn't say Pull Request, bc PRs are not part of a standard clone of RIOT (nor are say of the submodule) so you cannot simply change to a PR w/o changing the git-config of the submodule to also fetch PRs (which, again, is not done by default).

Reply:
OK, version then?

Reply:
Also I will fix the commit message